.github/workflows/test-daily.yaml

```diff
@@ -0,0 +1,26 @@
+name: Tests (DAILY)
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    # Daily at 03:00
+    - cron: '0 3 * * *'
+  workflow_call:
+
+jobs:
+  compile-tracee:
+    name: Compile Tracee through upstream libbpfgo
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout libbpfgo
+        uses: actions/checkout@v2
+        with:
+          repository: aquasecurity/libbpfgo
+          path: './'
+      - name: Install Dependencies
+        uses: aquasecurity/libbpfgo/.github/actions/build-dependencies@main
+        with:
+          go-version: 1.18
+      - name: Compile Tracee
+        run: |
+          sudo ./testing/compile-tracee.sh
```

Comment:
Thinking a bit more about this.. shouldn't we have a script as `./tests/libbpfgo.sh` in tracee branch and test latest libbpfgo using that script ? And then we remove "tracee" tests from libbpfgo. If we think right, it makes no sense to tie "libbpfgo" tests to tracee (or any other dependant project would need/deserve to be there as a test as well). WDYT ?

Reply:
I completely agree. I'm going to make the required changes to this. Later I'll remove the old test from libbpfgo. :+1: